saleor/core/utils/__init__.py

```diff
@@ -156,3 +157,19 @@ def generate_unique_slug(
 def delete_versatile_image(image):
     image.delete_all_created_images()
     image.delete(save=False)
+
+
+def graphql_id(instance):
+    class_name = instance.__class__.__name__
+    if instance.id is None:
+        return None
+    return graphene.Node.to_global_id(class_name, instance.id)
+
+
+def get_id_from_graphql_id(_id):
```

Comment:
Why not `id` instead of `_id`?

Reply:
And do we really need this method? Cannot we use `from_global_id_or_error` from `saleor/saleor/graphql/core/utils/__init__.py`?

Reply:
Fixed but still need additional function. It uses `from_global_id_or_error` but I need to return None in some scenarios. To avoid checking payload XX times in the logic that's why I've created such a function. Also moved to `saleor/saleor/graphql/core/utils/__init__.py` as they are part of grapqhl. [code](https://github.com/saleor/saleor/pull/9388/commits/723baa75e4df8f222e86ec5895acf1995f57cbba#diff-49b45f224daeaed23cfc4a19f956c588071fae3f97494be88617f04894be099bR186)